tests/test_vcr_cleaner.py

```diff
@@ -0,0 +1,50 @@
+
+# Python imports
+import datetime
+import gzip
+import os
+
+# Library imports
+import jwt
+import pytest
+import requests
+import vcr
+
+# VCRpy Cleaner imports
+from vcr_cleaner import CleanYAMLSerializer, cleaner, cassette
+from cleaners.jwt_token import clean_token
+
+# To record, `export VCR_RECORD=True`
+
+CASSETTE_ENDPOINT = 'https://cybersecurity.illinois.edu'
+
+# TODO: Add unit tests. 
+
+# TODO: Think about how to get rid of need 
+# to export VCR_RECORD and deleting the YAML
+def test_with_vcr(cassette):
```

Comment:
This whole test is an integration test at the moment, as is more important as documentation of how to use this library than as a test.

Should we move this into a 'integration' tests folder, or try harder to make it not read the file on the drive?

Reply:
Update per discussion - let's put a # TODO: test_with_vcr should become a DocTest(s) on this for later when we're building the documentation.